frontend/src/views/Join.tsx

```diff
@@ -129,6 +129,7 @@ function JoinGamePage() {
               setFocusInput(false);
             }}
             onKeyPress={(event) => {
+              setError('')
```

Comment:
What's the thinking here? -- when the user starts typing a username or `roomId` again, to remove the error if one is displayed? (Same question for the rest of these same changes.)

Reply:
Yeah that's what I was mainly thinking, so that once they start typing again (or submit/click a button again for the other places) the error would go away without having to refresh - open to reverting this given how small a change it is

Reply:
Nah, this seems like a good change, let's keep it.